site/source/tutorials/TypeScript_and_Dojo_2.md

```diff
@@ -0,0 +1,60 @@
+
+In addition to supporting team-coordination, the ability to describe a data-structure's expected *shape* clarifies the expected fields within [object literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#Object_literals). This is especially important when creating applications that interact with a server to load, manipulate, and save large amounts of data. Since most communication between Dojo 2 applications and servers is done using the [JSON protocol](http://www.json.org/), the ability to define interfaces for the messages flowing back and forth greatly simplifies the processing of that data.
+
+### The compiler
+
+Due to the complicated suite of applications within many enterprises, technology upgrades are often adopted slowly. This is necessary due to the large amount of testing that is required to ensure that the new technology does not break existing applications. This has historically hindered enterprise developers from accessing the latest features for the languages that they work in. TypeScript takes advantage of the fact that it must be compiled into JavaScript to bridge this technology gap. TypeScript is continually updated to include the latest features that are coming to the JavaScript language. The compiler then emits JavaScript that converts those features into language structures that are compatible with existing versions of the JavaScript specification, often back to ECMAScript 3. This brings the best of both worlds to the enterprise - they are able to maintain a carefully planned upgrade path for their web browsers and other technology while allowing their developers to take advantage of the latest features of the language.
+
+## Summary
+
+TypeScript's type system has allowed powerful development aids to be created such as refactoring tools and autocompletion support. This enables enterprise developers to focus on writing clean, efficient code without constantly having to lookup information on how to use existing APIs.
+
+Supporting multi-team projects has always been challenging in JavaScript due to its dynamic, untyped nature. By utilizing TypeScript interfaces, Dojo 2 projects can be divided in discrete domains that are bounded by mutually understood interfaces that define the behaviors and data structures. Within each domain, a team can write the code that they need without having to be concerned that they might be causing another break another team's code.
```

Comment:
A lot of the content in this article is overly wordy, let's get to the point. For example, " without having to be concerned that they might be causing another break another team's code." -> " without being concerned that they could break another team's source code."